### ISSUE TITLE
webplayer 404

### ISSUE THREAD

**Issue by snowboarder04** [podlove/podlove-publisher#1023]

**Comment by eteubert** [comment]
`/podlove/file/23/s/webplayer/c/episode/Ep0-test.m4a` is a tracking URL that redirects to the actual file.

You can check the tracking settings page for issues (`Expert Settings -> Tracking`) or deactivate it there if you don't need it.

**Comment by snowboarder04** [comment]
Turning off download counting in that settings panel appears to fix the problem. I would quite like to have this ability though - is there a way to resolve the root issue so that I'm able to? Thanks

**Comment by eteubert** [comment]
There's a debug section at `Expert Settings -> Tracking`, what does it say? Might be a redirect issue or conflicting server config. Hard to say since you didn't provide your URL.

**Comment by snowboarder04** [comment]
Thanks for your help so far! The debug section shows empty - it shows the section title but there is nothing beneath it except the 'Save Changes' button. Logging is enabled in the 'Modules' section. The Support tab debug log seems to have stopped logging approx 12 hours ago - I can't think why.

Both the site and the media subdomain are serving traffic over SSL with the "Check for Assets with SSL-peer-verification." checkbox set to off and "Website Protocol" set to option 2 (Website is delivered by HTTPS - including podcast feeds).

The only thing I can see in the nginx logs is that directory indexes of the podcast media directory is forbidden.

**Comment by snowboarder04** [comment]
Ok, so after much stracing (running php under fast_cgi != fun debugging), I figured it out it was a setting that was causing the trouble.

Under "Episode Assets" I had set the "Downloadable" option for each of my feeds to unchecked (I don't want visitors downloading the files, instead, I'd prefer they just to listen to them on the page). In hindsight, I did find it odd that the 'Files' menu was appearing under the player, indicating to the visitor that the files can be pulled. Is the logic between listening in-player and direct-download broken?

Can I ask what the function of the 'Downloadable' option here is for and if this isn't expected behaviour, perhaps I've caught a bug? Thanks!

**Comment by eteubert** [comment]
Right, that setting is probably misnamed. It should maybe be called "publicly accessible" instead. It was created for assets that you want to keep track but not expose to listeners, for example raw flac recording files.

If you don't want people to use subscription feeds, don't create feeds. A feed with non-downloadable files is pointless.
